templates/terraform/examples/cloud_run_service_multiple_environment_variables.tf.erb

```diff
@@ -0,0 +1,25 @@
+resource "google_cloud_run_service" "<%= ctx[:primary_resource_id] %>" {
+  name     = "<%= ctx[:vars]['cloud_run_service_name'] %>"
+  location = "us-central1"
+
+  template {
+    spec {
+      containers {
+        image = "gcr.io/cloudrun/hello"
+      }
+      env { 
```

Comment:
I think these actually should be inside of the containers block- running the example as-is gives the error `Unsupported block type: Blocks of type "env" are not expected here.`

Reply:
Whelp, sorry about that, I copied my own example wrong ><